atomicfu/build.gradle.kts

```diff
@@ -0,0 +1,382 @@
+import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
+import org.jetbrains.kotlin.gradle.targets.js.dsl.ExperimentalWasmDsl
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.jetbrains.kotlin.gradle.tasks.KotlinJvmCompile
+
+plugins {
+    id("kotlin-multiplatform-conventions")
+    id("kotlin-multiplatform-publish-conventions")
+}
+
+kotlin {
+
+    // JS -- always
+    js(IR) {
+        moduleName = "kotlinx-atomicfu"
+        // TODO: commented out because browser tests do not work on TeamCity
+        // browser()
+        nodejs()
+    }
+
+    // JVM -- always
+    jvm()
+
+    // Wasm -- always
+    @OptIn(ExperimentalWasmDsl::class)
+    wasmJs {
+        nodejs()
+    }
+
+    @OptIn(ExperimentalWasmDsl::class)
+    wasmWasi {
+        nodejs()
+    }
+
+    sourceSets {
+        commonMain.dependencies {
+            implementation("org.jetbrains.kotlin:kotlin-stdlib") {
+                version {
+                    prefer(libs.versions.kotlin.get())
+                }
+            }
+        }
+        commonTest.dependencies {
+            implementation("org.jetbrains.kotlin:kotlin-test-common")
+            implementation("org.jetbrains.kotlin:kotlin-test-annotations-common")
+        }
+
+        val jsAndWasmSharedMain by creating {
+            dependsOn(commonMain.get())
+        }
+
+        jsMain {
+            dependsOn(jsAndWasmSharedMain)
+            dependencies {
+                compileOnly("org.jetbrains.kotlin:kotlin-dom-api-compat")
+            }
+        }
+
+        jsTest {
+            dependencies {
+                implementation("org.jetbrains.kotlin:kotlin-test-js")
+            }
+        }
+
+        wasmJsMain {
+            dependsOn(jsAndWasmSharedMain)
+        }
+
+        wasmJsTest {
+            dependencies {
+                implementation("org.jetbrains.kotlin:kotlin-test-wasm-js")
+            }
+        }
+
+
+        wasmWasiMain {
+            dependsOn(jsAndWasmSharedMain)
+        }
+
+        wasmWasiTest {
+            dependencies {
+                implementation("org.jetbrains.kotlin:kotlin-test-wasm-wasi")
+            }
+        }
+
+        jvmTest {
+            dependencies {
+                implementation("org.jetbrains.kotlin:kotlin-reflect")
+                implementation("org.jetbrains.kotlin:kotlin-test")
+                implementation("org.jetbrains.kotlin:kotlin-test-junit")
+                implementation(libs.junit.junit)
+            }
+        }
+    }
+}
+
+// Support of all non-deprecated targets from the official tier list: https://kotlinlang.org/docs/native-target-support.html
```

Comment:
Even tho `linuxArm32Hfp` is deprecated, most of the other kotlinx libraries have added it anyway.
Could atomicfu do the same for consistency?

Reply:
Thank you, yes, I'll add this target in a separate PR

Reply:
https://github.com/Kotlin/kotlinx-atomicfu/pull/455